### PR TITLE
Updated config.json.example

### PR DESCRIPTION
**Pull request by Courty40**
added extra control categories to match the membership systems

### CODE REVIEW

**Comment by unknowndomain** [comment]
This isn't intended use for control-node, it really needs to be replaced with something that works with the event system in the membership system, a fair chunk of work has gone into this.

We should talk about what you are trying to do and what help you need.

**Comment by Courty40** [comment]
Hi Tom,
We're just trying to get basic tool control working again for the time being (laser + a couple of other bits to test the dusty system).   Be good to get a better view of what's there and what we could do with it.

Courty

**Comment by unknowndomain** [comment]
I don't think this is the best way because it will result in more notifications in the event log and really a solution to replace it is needed.

When can we meet?

**Comment by Courty40** [comment]
I'll be there Thursday. if you can make it over it would be good to catch up !

**Comment by unknowndomain** [comment]
I'm not a member any more but sure, however we should involve Jonathan too.

Could you advise you've decided to do and the state of play is with things before hand, last I heard Pip and Joe were building a system using CAN.

**Comment by ghost** [comment]
lol 

3 years

**Comment by ghost** [comment]
no love for the necromancy?

**Comment by unknowndomain** [comment]
I feel that this ticket PR can be closed unresolved.

control-node exists to broker the communication between dumb Arduino based microcontrollers and the HTTPS membership system.

None of the tool control solutions I've seen discussed since that projected was taken over from me by Pip, then Jonathan, then Courty have been based around lower power MCUs like doorbot and shutterbot.

I know that Jonathan's system was based around a RS485 to HTTPS broker to send requests to the membership-system directly, and Courty was using MQTT broker to send requests to the membership-system.

To summarise, this PR is redundant at this point because the tool control project moved beyond it and while I am unsure the state of that project now, if a new project was to be intitiated a base requirement surely should be the ability to issue requests to the membership-system directly or via a new broker.
